docs/_markbind/navigation/userGuideSections.md

```diff
@@ -0,0 +1,11 @@
+<markdown class="website-content">
+- ## Sections
+- <a href="{{baseUrl}}/userGuide/userQuickStart.html">Quick Start</a>
+- <a href="{{baseUrl}}/userGuide/developingASite.html">Developing a Site</a>
+  - <a href="{{baseUrl}}/userGuide/siteConfiguration.html">Site Configuration</a>
+- <a href="{{baseUrl}}/userGuide/contentAuthoring.html">Content Authoring</a>
+  - <a href="{{baseUrl}}/userGuide/includingContents.html">Including Contents</a>
```

Comment:
A new page "Known Problems" have been added in a recent PR, do add that in as well.